examples/simple_calculator/src/aiq_simple_calculator/configs/config-tracing.yml

```diff
@@ -38,6 +38,14 @@ general:
         method: POST
         description: Gets the current time
         function_name: current_datetime
+    cross_origin_resource_sharing:
+      allow_origins:
+      allow_origin_regex:
+      allow_methods:
+      allow_headers:
+      allow_credentials:
+      expose_headers:
+      max_age:
```

Comment:
Remove the default values here unless were actually setting something. Lets just keep

```
allow_origins: ["*"]
```